Userland/Demos/Starfield/Starfield.cpp

```diff
@@ -85,9 +85,19 @@ void Starfield::mousedown_event(GUI::MouseEvent&)
     GUI::Application::the()->quit();
 }
 
-void Starfield::keydown_event(GUI::KeyEvent&)
+void Starfield::keydown_event(GUI::KeyEvent& event)
 {
-    GUI::Application::the()->quit();
+    switch (event.key()) {
+    case Key_Plus:
+        m_speed++;
+        break;
+    case Key_Minus:
+        if (--m_speed < 1)
```

Comment:
`m_speed` is now unsigned, so you'll need to check this differently

Reply:
The only possible alternative is `if (--mspeed == 0)`. Is it worth it?
The current code still works correctly.

Reply:
Well this makes no sense, an unsigned number will never be `< 0`. Please update the check so that it makes sense :)

Reply:
Oh I see, it's `< 1` and not 0 :sweat_smile: - nevermind!